tests/src/test/testSuite.ts

```diff
@@ -40,43 +41,54 @@ const testSuites_1_2 = [
   fdc3AddContextListener_1_2,
   fdc3AddIntentListener_1_2,
   fdc3JoinChannel_1_2,
+  fdc3FindIntentsByContext_1_2,
+];
+
+const advancedSuite_1_2 = [
   fdc3Open_1_2,
   fdc3Broadcast_1_2,
   fdc3FindIntent_1_2,
   fdc3RaiseIntent_1_2,
   fdc3RaiseIntentForContext_1_2,
-  fdc3FindIntentsByContext_1_2,
 ];
 
-const testSuites_2_0 = [
-  // fdc3AddContextListener_2_0,
-  // fdc3AddIntentListener_2_0,
-  fdc3GetInfo_2_0,
-  fdc3getAppMetadata_2_0,
-  fdc3GetInstances_2_0,
-  fdc3GetUserChannels_2_0,
-  fdc3Broadcast_2_0,
-  // fdc3GetCurrentChannel_2_0,
-  // fdc3GetOrCreateChannel_2_0,
-  // fdc3JoinChannel_2_0,
-  // fdc3LeaveCurrentChannel_2_0,
-  // fdc3FindIntent_2_0,
-  // fdc3Open_2_0,
-  // fdc3RaiseIntent_2_0,
-  // fdc3RaiseIntentForContext_2_0,
-  // fdc3FindIntentsByContext_2_0,
-];
+
+const allSuites_1_2 = [...basicSuite_1_2, ...advancedSuite_1_2];
+
+export const packs: { [index: string]: (() => Suite)[] } = {
+  All_1_2: allSuites_1_2,
+  Basic_1_2: basicSuite_1_2,
+  Advanced_1_2: advancedSuite_1_2,
+  fdc3AddContextListener_1_2: [fdc3AddContextListener_1_2],
+  fdc3Broadcast_1_2: [fdc3Broadcast_1_2],
+  fdc3FindIntent_1_2: [fdc3FindIntent_1_2],
+  fdc3Open_1_2: [fdc3Open_1_2],
+  fdc3RaiseIntent_1_2: [fdc3RaiseIntent_1_2],
+  fdc3RaiseIntentForContext_1_2: [fdc3RaiseIntentForContext_1_2],
+  fdc3AddIntentListener_1_2: [fdc3AddIntentListener_1_2],
+  fdc3GetCurrentChannel_1_2: [fdc3GetCurrentChannel_1_2],
+  fdc3GetInfo_1_2: [fdc3GetInfo_1_2],
+  fdc3GetOrCreateChannel_1_2: [fdc3GetOrCreateChannel_1_2],
+  fdc3GetSystemChannels_1_2: [fdc3GetSystemChannels_1_2],
+  fdc3JoinChannel_1_2: [fdc3JoinChannel_1_2],
+  fdc3LeaveCurrentChannel_1_2: [fdc3LeaveCurrentChannel_1_2],
+  fdc3FindIntentsByContext_1_2: [fdc3FindIntentsByContext_1_2],
+};
+
+export function getPackNames(): string[] {
+  return Object.keys(packs);
+}
 
 /**
  * Intended for running tests in container with results shown
  * in HTML page
  */
-export const executeTestsInBrowser = async (fdc3Version: string) => {
+<<<<<<< HEAD
+export const executeTestsInBrowser = (pack: string) => {
   (mocha as any).timeout(constants.TestTimeout);
-  if (fdc3Version === "v1.2") {
-    testSuites_1_2.forEach((suite) => suite());
-  } else if (fdc3Version === "v2.0") {
-    testSuites_2_0.forEach((suite) => suite());
-  }
+  packs[pack]
+    .sort((s1, s2) => s1.name.localeCompare(s2.name))
```

Comment:
`.sort((s1, s2) => s1.name.localeCompare(s2.name))` This doesn't do anything

Reply:
this was already there!